migtests/tests/import-txt-file-test/SMSA.txt

```diff
@@ -0,0 +1,60 @@
+Akron	OH	27	71	59	36	921.87	11.4	3243	8.8	42.6	660328	3.34	29560	21	15	59	15
```

Comment:
Consider using delimiter other than tab (Something like a `|` character).
Add some rows where the delimiter itself appears as a part of a string and thus you will have to escape it.